### PR TITLE
Diagnose Disk usage checks

### PR DESCRIPTION
**Pull request by sgmiller**

### CODE REVIEW

File: command/operator_diagnose.go [hunk 1]
```diff
@@ -288,4 +290,37 @@ func (c *OperatorDiagnoseCommand) offlineDiagnostics(ctx context.Context) error
...
+		for _, partition := range partitions {
+			for _, exc := range partitionExcludes {
+				if strings.HasPrefix(partition.Mountpoint, exc) {
+					continue partLoop
+				}
+			}
+			usage, err := disk.Usage(partition.Mountpoint)
+			testName := "disk-usage: " + partition.Mountpoint
+			if err != nil {
+				diagnose.Warn(ctx, fmt.Sprintf("could not obtain partition usage for %s", partition.Mountpoint))
```
Comment (swayne275) [inline command/operator_diagnose.go:311]: do we want to report the error in here? i think that may be helpful?

**Review by HridoyRoy — CHANGES_REQUESTED** [review]

File: command/operator_diagnose.go [hunk 2]
```diff
@@ -288,4 +290,37 @@ func (c *OperatorDiagnoseCommand) offlineDiagnostics(ctx context.Context) error
 		}
 		return nil
 	})
+
+	diagnose.Test(ctx, "disk-usage", func(ctx context.Context) error {
```
[the remark this answers is not in the document]
Comment (sgmiller) [reply command/operator_diagnose.go:320]: Sure

File: command/operator_diagnose.go [hunk 3]
```diff
@@ -2,6 +2,8 @@ package command
 
 import (
 	"context"
+	"fmt"
+	"github.com/shirou/gopsutil/disk"
```
[the remark this answers is not in the document]
Comment (sgmiller) [reply command/operator_diagnose.go]: We already had it as a dependency, so in fact nothing new here.  It's used by the hostinfo endpoint/helper.

File: command/operator_diagnose.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (sgmiller) [reply command/operator_diagnose.go]: Yep.

**Review by HridoyRoy — APPROVED** [review]
lgtm!

File: command/operator_diagnose.go (hunk 1 above)
Comment (sgmiller) [reply command/operator_diagnose.go]: Yep, good call

**Pull request merged**